nano/rpc/rpc_handler.cpp

```diff
@@ -4137,6 +4137,27 @@ void nano::rpc_handler::wallet_republish ()
 	response_errors ();
 }
 
+void nano::rpc_handler::wallet_seed ()
```

Comment:
Implementation 2?

Reply:
rebase-fail (there was some duplication in the rpc refactoring I think. I believe that has been fixed, so I'll rebase it and fix)